_layouts/blog-post.html

```diff
@@ -0,0 +1,17 @@
+---
+layout: simple
+---
+
+<section class="py-5">
+  <div class="container text-center">
+    <h2>{{ page.title }}</h2>
+    <p>{{ page.printDate }}, by <a href="#">{{ page.author }}</a></p>
```

Comment:
I don't think we'll have author links, so the author name doesn't need to be a link.

Reply:
I've removed this.